apps/ima/fields/index.js

```diff
@@ -2,9 +2,37 @@
 
 module.exports = {
   'who-are-you': {
-    isPageHeading: true,
     mixin: 'radio-group',
     options: ['person-named', 'has-legal-representative', 'someone-else'],
+    validate: 'required',
+    legend: {
+      className: 'visuallyhidden'
+    }
+  },
+  image: {
+    mixin: 'input-file',
+    labelClassName: 'visuallyhidden'
+  },
+  'declaration-immigration-adviser': {
+    isPageHeading: false,
+    mixin: 'radio-group',
+    labelClassName: 'visuallyhidden',
+    options: [{
+      value: 'yes'
+    }, {
+      value: 'no'
+    }],
+    validate: 'required'
+  },
+  'declaration-someone-else': {
+    isPageHeading: false,
+    mixin: 'radio-group',
+    labelClassName: 'visuallyhidden',
+    options: [{
+      value: 'yes'
+    }, {
+      value: 'no'
+    }],
     validate: 'required'
   }
```

Comment:
Validation messages not showing on top of screen or above mandatory fields if left blank. Same goes for the other mandatory fields.

Reply:
Validation added with messages for all mandatory fields.